icu4c/source/i18n/unicode/unum.h

```diff
@@ -302,7 +302,24 @@ typedef enum UNumberFormatRoundingMode {
       * ROUND_UNNECESSARY reports an error if formatted result is not exact.
       * @stable ICU 4.8
       */
-    UNUM_ROUND_UNNECESSARY
+    UNUM_ROUND_UNNECESSARY,
+#ifndef U_HIDE_DRAFT_API
+    /**
+     * Round ties toward the odd number.
```

Comment:
Small preference for third person:

```suggestion
     * Rounds ties toward the odd number.
```